contrib/process/runnable.go

```diff
@@ -33,5 +33,5 @@ func (p *process) Run(ctx context.Context) error {
 	defer cancel()
 
 	<-ctx.Done()
-	return ctx.Err()
+	return nil
```

Comment:
I'm not sure we want to just return `nil` when we've received a signal that this runnable is tracking e.g. an instance of this runnable could be used to monitor signals that we only expect to be fired in an error scenario, and thus we'd want some error returned from this runnable.

I think maybe we'd want some way to create this runnable such that we can set the cause on the ctx when it is canceled or if the cause is already there we want to propagate it in our code that uses the runnable.

Reply:
Discussed offline, and not returning an error seems like the semantically correct thing to do here